### PR TITLE
Update virtualbox-beta install

### PR DESCRIPTION
**Pull request by commitay**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [ ] Named the cask according to the [token reference].
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked there are no [open pull requests] for the same cask.
- [ ] Checked the cask was not [already refused].
- [ ] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask-versions/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask-versions/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

Goes with https://github.com/Homebrew/homebrew-cask/pull/72804.

Update to match `virtualbox`.

### CODE REVIEW

File: Casks/virtualbox-beta.rb [hunk 1]
```diff
@@ -7,27 +7,55 @@
...
                      },
-            pkgutil: 'org.virtualbox.pkg.*'
+            pkgutil: 'org.virtualbox.pkg.*',
+            delete:  '/usr/local/bin/vboximg-mount'
 
   zap trash: [
                '/Library/Application Support/VirtualBox',
                '~/Library/Application Support/com.apple.sharedfilelist/com.apple.LSSharedFileList.ApplicationRecentDocuments/org.virtualbox.app.virtualbox.sfl*',
                '~/Library/Application Support/com.apple.sharedfilelist/com.apple.LSSharedFileList.ApplicationRecentDocuments/org.virtualbox.app.virtualboxvm.sfl*',
+               '~/Library/VirtualBox',
```
Comment (reitermarkus) [inline Casks/virtualbox-beta.rb]: This should still be sorted alphabetically.

File: Casks/virtualbox-beta.rb [hunk 2]
```diff
@@ -7,27 +7,55 @@
...
+                   'choiceAttribute'  => 'selected',
+                   'attributeSetting' => 1,
+                 },
+                 {
+                   'choiceIdentifier' => 'choiceVBoxCLI',
+                   'choiceAttribute'  => 'selected',
+                   'attributeSetting' => 1,
+                 },
+                 {
+                   'choiceIdentifier' => 'choiceOSXFuseCore',
```
Comment (reitermarkus) [inline Casks/virtualbox-beta.rb]: Why not install this? Do we need to depend on the `osxfuse` cask instead?

File: Casks/virtualbox-beta.rb (hunk 1 above)
Comment (commitay) [reply Casks/virtualbox-beta.rb]: I'll revert this and sort the `virtualbox` cask alphabetically.

File: Casks/virtualbox-beta.rb (hunk 2 above)
Comment (commitay) [reply Casks/virtualbox-beta.rb]: See https://github.com/Homebrew/homebrew-cask/pull/72804.

File: Casks/virtualbox-beta.rb (hunk 1 above)
Comment (commitay) [reply Casks/virtualbox-beta.rb]: Fixed.

**Pull request merged**